src/pages/DatasetStatistics.jsx

```diff
@@ -78,52 +78,28 @@ export default function DatasetStatistics(props) {
               <div style={{ ...Styles.MINOR_HEADER, paddingLeft: '10px' }}>Dataset Description:</div>
               {LINE}
               <div style={{ fontSize: Theme.font.size.small, padding: '1rem' }}>
-                {get(
-                  find(dataset?.properties, (p) => {
-                    return p.propertyName === 'Description';
-                  }),
-                  'propertyValue',
-                  ''
-                )}
+                {extract('Dataset Description') || dataset?.description || dataset?.study?.description }
```

Comment:
Should we add a default value, such as "N/A", for situations where no values are provided, or should we gray out the properties? Just a thought. No need to implement it, of course.